content/en/docs/concepts/workloads/controllers/_index.md

```diff
@@ -3,3 +3,51 @@ title: "Workload Resources"
 weight: 20
 ---
 
+Kubernetes provides several built-in APIs for declarative management of your
+{{< glossary_tooltip text="workloads" term_id="workload" >}}
+and the components of those workloads.
+
+Ultimately, your applications run as containers inside
+{{< glossary_tooltip term_id="Pod" text="Pods" >}}; however, managing individual
+`Pods` would be a lot of effort. For example, if a `Pod` fails, you probably want to
```

Comment:
I don't think we have `Pods` anywhere in the docs. We talk about `Pod`, which is a API resource kind. We also talk about `pods`, which is a collection of Pod objects. We don't have `Pods`. WDYT?